app/assets/stylesheets/items.scss

```diff
@@ -1,3 +1,14 @@
 // Place all the styles related to the items controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+.summary{
+  font-size: 25px;
```

Comment:
Properties should be ordered background-color, border-color, border-radius, border-style, display, font-size, margin, padding